scripts/common/info-view.ts

```diff
@@ -5,24 +5,23 @@ import {displayWarning} from './uiElements'
 //  * @param {int} price - The price of the ticket to display.
 //  * @param {string} venmoRecipient - The reciepient of the venmo transaction to display.
 //  */
-// const generateVenmoInfo = function generateVenmoInfoForView(venmoRecipient:string) {
-//     const venmoLinkButton = document.querySelector('.open-venmo') as HTMLButtonElement;
-//     const venmoLinkPath = `venmo://paycharge?txn=pay&recipients=${venmoRecipient}&note=Eweek Banquet`;
+const generateVenmoInfo = function generateVenmoInfoForView(venmoRecipient:string) {
+    const venmoLinkButton = document.querySelector('.open-venmo') as HTMLButtonElement;
+    const venmoLinkPath = `venmo://paycharge?txn=pay&recipients=${venmoRecipient}&note=Eweek Banquet`;
```

Comment:
Does having a space character in the URL here work? You may need to use an URL escaped version of the space character.